account_asset_management/models/account_move.py

```diff
@@ -256,6 +256,13 @@ def _expand_asset_line(self):
                 aml = self.with_context(check_move_validity=False)
                 qty = self.quantity
                 name = self.name
-                aml.write({"quantity": 1, "name": "{} {}".format(name, 1)})
+                price = self.price_unit
+                aml.write(
+                    {
+                        "quantity": 1,
+                        "name": "{} {}".format(name, 1),
+                        "price_unit": price,
+                    }
+                )
```

Comment:
Always document in code "weird" things done:

```suggestion
                aml.write(
                    {
                        "quantity": 1,
                        "name": "{} {}".format(name, 1),
                        # Make sure the price is not changed, like with account_invoice_pricelist
                        "price_unit": self.price_unit,
                    }
                )
```